src/etc/inc/dyndns.class

```diff
@@ -702,6 +703,19 @@
 					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
 					curl_setopt($ch, CURLOPT_URL, $server . '?domains=' . str_replace('.duckdns.org', '', $this->_dnsHost) . '&token=' . $this->_dnsUser);
 					break;
+				case 'googledomains':
+					if ($this->_dnsVerboseLog)
+						log_error("Google Domains: ({$this->_dnsHost}): DNS update() starting.");
+					$server = "https://domains.google.com/nic/update";
+					$post_data['hostname'] = $this->_dnsHost;
+					$post_data['myip'] = $this->_dnsIP;
+					$post_data['offline'] = 'no';
+					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
```

Comment:
After removing it and testing, I now get this error:
`opnsense: /services_dyndns_edit.php: Curl error occurred: SSL certificate problem: unable to get local issuer certificate`

Google recently began operating its own CA, which it now uses for domains.google.com. The root certificate does not appear to be in `/usr/local/share/certs/ca-root-nss.crt`, although I believe the root cert is cross-signed and I'm not 100% sure that is where curl is looking for it.

On the other hand, `CURLOPT_SSL_VERIFYPEER` is set to false in every other case except the custom one, where the user has an option in the GUI to set it to true. So this may be part of a larger issue in the codebase.

It looks like we may need to explicitly set `CURLOPT_CAINFO` to tell curl where to find certificates, if there is not a default that points to something useful.